modules/react/dialog/lib/DialogCard.tsx

```diff
@@ -5,11 +5,11 @@ import {Popup, PopupModelContext} from '@workday/canvas-kit-react/popup';
 
 import {useDialogCard} from './hooks';
 
-export interface DialogPopupProps extends ExtractProps<typeof Popup.Popper, never> {}
+export interface DialogCardProps extends ExtractProps<typeof Popup.Card, never> {}
```

Comment:
Probably a copy/paste error. The interface is updated to reflect the correct props and naming convention.